### PR TITLE
[KAFKA] Change repo name to kafka for compatability

### PR DESCRIPTION
**Pull request by mbdcos**
Update universe and yaml to change repo name to kafka. 

Originally it was *kafka*. We later set its name to *apache-kafka* (to differentiate from confluent-kafka).

Role, principle, etc are based on this name. 

To be compatible with all other frameworks and to have the same name with the framework directory (frameworks/kafka), want to change name back to *kafka*.

PS: Description is always "Apache Kafka".

### CODE REVIEW

File: frameworks/kafka/src/main/dist/svc.yml [hunk 1]
```diff
@@ -17,17 +19,22 @@ pods:
           broker:
             port: {{BROKER_PORT}}
             env-key: KAFKA_BROKER_PORT
+            vip:
+              prefix: broker
+              port: 9092
           {{#BROKER_JMX_PORT}}
           jmx:
-            port: {{JMX_PORT}}
+            port: {{BROKER_JMX_PORT}}
```
Comment (gabrielhartmann) [inline frameworks/kafka/src/main/dist/svc.yml]: @triclambert It's my understanding we want to drop JMX stuff, right?

File: frameworks/kafka/universe/marathon.json.mustache [hunk 2]
```diff
@@ -54,18 +54,19 @@
     "CONFIG_TEMPLATE_PATH": "kafka-scheduler",
     "KAFKA_VERSION_PATH": "kafka_2.11-0.10.1.0",
     "MESOS_ZOOKEEPER_URI": "master.mesos:2181",
-    {{#brokers.jmx.enable}}
-     "TASKCFG_ALL_KAKFA_JMX_OPTS": "-Dcom.sun.management.jmxremote={{brokers.jmx.remote}} -Dcom.sun.management.jmxremote.registry.ssl={{brokers.jmx.remote_registry_ssl}} -Dcom.sun.management.jmxremote.ssl={{brokers.jmx.remote_ssl}} -Dcom.sun.management.jmxremote.ssl.need.client.auth={{brokers.jmx.remote_ssl_need_client_auth}} -Dcom.sun.management.jmxremote.authenticate={{brokers.jmx.remote_authenticate}} {{brokers.additional_kafka_jmx_opts}}"
-     "BROKER_JMX_PORT": "{{brokers.jmx.remote_port}}",
-     {{/brokers.jmx.enable}}
 
+    {{#brokers.jmx.enable}}
```
Comment (gabrielhartmann) [inline frameworks/kafka/universe/marathon.json.mustache]: Should probably remove this as well pending @triclambert's comment

Comment (mbdcos) [inline frameworks/kafka/universe/marathon.json.mustache]: @triclambert I would prefer keeping JMX now. It is one less item to document for upgrade. It exist already in dcos-kafka-service.

File: frameworks/kafka/universe/config.json [hunk 3]
```diff
@@ -6,9 +6,9 @@
         "description": "DC/OS service configuration properties",
         "properties":{
           "name" : {
-            "description":"Apache Kafka",
+            "description":"Apache Kafka (apache-kafka)",
```
Comment (gabrielhartmann) [inline frameworks/kafka/universe/config.json]: Please remove the parenthetical

**Pull request merged**